appcrashtracker/src/main/java/com/org/appcrashtracker/ExceptionHandler.java

```diff
@@ -1,124 +1,45 @@
 package com.org.appcrashtracker;
 
-import com.org.appcrashtracker.utils.ApplicationInfoUtils;
-import com.org.appcrashtracker.utils.BatteryUtils;
-import com.org.appcrashtracker.utils.NetworkUtils;
+import com.org.appcrashtracker.data.ExceptionData;
 import ohos.aafwk.ability.Ability;
 import ohos.aafwk.content.Intent;
 import ohos.aafwk.content.Operation;
 import ohos.app.Context;
 import ohos.app.Environment;
-import ohos.bundle.BundleInfo;
 import ohos.global.resource.NotExistException;
-import ohos.global.resource.ResourceManager;
 import ohos.global.resource.WrongTypeException;
 import ohos.hiviewdfx.HiChecker;
 import ohos.hiviewdfx.HiLog;
 import ohos.hiviewdfx.HiLogLabel;
-import ohos.system.DeviceInfo;
 import ohos.utils.zson.ZSONException;
 import ohos.utils.zson.ZSONObject;
 
 import java.io.*;
-import java.util.Locale;
-
-import static com.example.appcrashtracker.ResourceTable.*;
-import static com.org.appcrashtracker.utils.MemoryUtils.*;
 
 public class ExceptionHandler implements java.lang.Thread.UncaughtExceptionHandler {
     private final Ability ability;
-    private ZSONObject jObjectData;
-    private String abilityName;
     private final HiLogLabel label = new HiLogLabel(HiLog.LOG_APP, 0x00201, getClass().getName());
-    private boolean className = false;
-    private boolean message = false;
-    private boolean localizedMessage = false;
-    private boolean causes = false;
-    private boolean stackTraceBoolean = false;
-    private boolean brandName = false;
-    private boolean deviceName = false;
-    private boolean sdkVersion = false;
-    private boolean release = false;
-    private boolean height = false;
-    private boolean width = false;
-    private boolean appVersion = false;
-    private boolean tablet = false;
-    private boolean deviceOrientation = false;
-    private boolean screenLayout = false;
-    private boolean vmHeapSize = false;
-    private boolean allocatedVmSize = false;
-    private boolean vmMaxHeapSize = false;
-    private boolean vmFreeHeapSize = false;
-    private boolean nativeAllocatedSize = false;
-    private boolean batteryPercentage = false;
-    private boolean batteryCharging = false;
-    private boolean batteryChargingVia = false;
-    private boolean sdCardStatus = false;
-    private boolean internalMemorySize = false;
-    private boolean externalMemorySize = false;
-    private boolean internalFreeSpace = false;
-    private boolean externalFreeSpace = false;
-    private boolean packageName = false;
-    private boolean networkMode = false;
-    private boolean country = false;
+    ExceptionData data = new ExceptionData();
```

Comment:
Should have private access modifier and only variable declaration should be there. Initialization must be in Constructor at L#67.